ONSdigital/eq-survey-runner
# Fixes 236 - refactor Metadata Store to parse dates on creation

### Changes

Fixes #236 - refactoring Meta Data store to parse dates on creation. If they don't parse correctly throw an invalid token exception. This will force the application to return a 403 forbidden.
### How to test
1. Go to the dev page
2. Enter an invalid date for either ref_p_start_date, ref_p_end_date or return_by
3. Check a 403 forbidden message is displayed
### Who can test

Anyone apart from @warrenbailey
